index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/3.9.1/chart.min.js"></script>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Stock Exchange</title><link rel="icon" type="image/x-icon" href="./img/productivity.png">
+    </head>
+  </head>
+ 
+  <body>
+    <!--<div class="page_container">
+      <div id="marquee" class="marquee"></div>
+      <h1>Nasdaq information for all</h1>
+      <h2> find your next stock investment:</h2>
+      <h4> Which company got your attention?</h4>
+      <div class="search_container">
+        <input type="search" id="companySearch" placeholder="Company name">        
+        <button type="button">Search</button>
+        <div id="loading" class="spinner-border d-none" role="status"></div>
+      </div>
+      <div id="form"></div>
+      <div id="show_result">
+      <ul id="results"></ul>
+      </div>
+    </div>-->
+    <div class="page_container">
+      <div id="marquee" class="marquee"></div>
+      <h1>Nasdaq information for all</h1>
+      <h2> find your next stock investment:</h2>
+      <h4> Which company got your attention?</h4>
+      <div id="form"></div>
+      <div id="results"></div>
+      </div>
+    <script type="module" src="js\Marquee.js"></script>
+    <script type="module" src="js\SearchResult.js"></script>  
+    <script type="module" src="js\SearchForm.js"></script>
+    <script type="module">
+      import {Marquee} from "./js/Marquee.js";
+      import {SearchForm} from "./js/SearchForm.js";
+      import {SearchResults} from "./js/SearchResult.js";
+
+      (async function () {
+        const marquee = new Marquee(document.getElementById('marquee'));
+        await marquee.load();
+        const form = new SearchForm(document.getElementById('form'));
+        form.addFormToPage();
+        const result = new SearchResults(document.getElementById('results'));
+
+        form.registerButtonOnClick(() => {result.renderResults()});
+       
+        //result.renderResults();
+       
+      })()
```

Comment:
It's a bad habit to write scripts here.
Please move it to a new js file and import that file here.